packages/ui/TabBar/index.tsx

```diff
@@ -1,6 +1,7 @@
 "use client";
 
 import { cn } from "@formbricks/lib/cn";
+import { env } from "@formbricks/lib/env";
```

Comment:
This should also not be imported in a client component once you move the API fetching to a server component